content/en/certificate.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Certificate
+status: Feedback Appreciated
+category: technology
+---
+
+
+## What it is
+
+A certificate — also often referred to as a digital certificate, public key certificate, or SSL certificate — is a technology used to verify the identity of an entity on the internet. 
+An entity can be the sender of an email or a website. Additionally, certificates encrypt communication between entities on the internet.
+
+
+## Problem it addresses
+
+Communication over the internet is unencrypted by default. 
+Consequently, any communication can potentially be intercepted, compromising sensitive information like usernames and passwords. 
+Think about your favorite coffee shop or a hotel lobby, where you connect to the available wifi router. 
+Someone could get between your device and the wifi router and get hold of your credentials or read your emails and chats.
+
+
+## How it helps
+
+Certificates are a technology that, on the one side, verifies that communication partners are who they say they are — 
+for example, an email sender or a website that you visit on your browser. 
+Modern email clients will notify you if a sender's identity is correct, as will the browsers (notice the little lock in front of the URL). 
+A certificate's trust derives from the certificate authority, often abbreviated as CA, that issues the certificate, guaranteeing that the certificate holder's identity is correct.
+
+On the other side, certificates encrypt communication between entities on the internet. 
+They contain a public key used to encrypt data, which can only be decrypted with a matching private key. 
+As the name indicates, public keys are shared publicly, while private keys are kept private. Take a website, for example. 
+Your browser encrypts data sent to the website using a public key, but only the website server knows the private key and can thus decrypt the data again. 
```

Comment:
Let me see if I understood correctly: the public key is able to encrypt but not decrypt (otherwise encrypting with a public key wouldn't make much sense as anyone could decrypt it again). To decrypt it you need the private key, is that correct?

If so, I think we need to clarify. In daily life, a key is used to close and open a door, so if it's public, it's available to anyone. If you need two keys, one to open and one to close, we need to say so. That's why I'm stumbling over this example and others might too.

Reply:
Exactly, like you explained :)

Reply:
Yes, you understood correctly :). I am happy for any suggestion to make it more clear :)

Reply:
How about this?

> On the other side, certificates encrypt communication between entities on the internet. Encryption and decryption happen with so-called keys. But unlike physical keys, certificates need two keys: a public one to encrypt data and a private one to decrypt it again. As the name indicates, public keys are shared publicly, while private keys are kept private. Take a website, for example. The browser encrypts data sent to the website using a public key and the website server holds the private key needed to decrypt it.

What I still don't get is: The public key is only for that website but, since it's used to encrypt data, they don't bother to make it private. Encryption through that public key can only be decrypted with that specific private key, right? If that's the case, maybe this is better?

>  On the other side, certificates encrypt communication between entities on the internet. Encryption and decryption happen with so-called keys. But unlike physical keys, certificates need two keys: a public one to encrypt data and a private one to decrypt it again. As the name indicates, public keys are shared publicly. Since they encrypt data, there is no need to hide it from the public. Private keys, on the other hand, are needed to decrypt data and are thus only held by those entities authorized to read it. Take a website, for example. The browser encrypts data sent to the website using a public key while the website server holds the private key needed to decrypt it.

Reply:
TBH I am not sure if we really should explain asymmetric encryption in that much detail, as the focus of this article is on certificates and not on how encryption works. I mean one must not necessarily understand how asymmetric encryption works in order to understand what certificates are, right? Can we not just simply say, that certificates can be used for asymmetric encryption because they include an encryption key? 

I would love to hear another opinion. @JasonMorgan @seokho-son @jihoon-seo, maybe. 

In case we want to keep the detailed explanation here a little modification to make it more accurate again :) 

`On the other side, certificates can be used to encrypt communication between entities on the internet. Encryption and decryption happen with so-called keys. But unlike physical keys, with this kind of encryption/decryption, two keys are needed: a public one to encrypt data and a private one to decrypt it again. As the name indicates, public keys are shared publicly. Since they encrypt data, there is no need to hide it from the public. Private keys, on the other hand, are needed to decrypt data and are thus only held by those entities authorized to read it. Take a website, for example. The browser encrypts data sent to the website using a public key (which is included in the certificate of that website) while the website server holds the private key needed to decrypt it.`

Reply:
Looks good to me. At least I learned a lot from these back and forth, so thank you!